src/main/java/core/basesyntax/model/FruitNegotiation.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax.model;
+
+import core.basesyntax.exception.FruitStoreException;
+
+public class FruitNegotiation {
```

Comment:
```suggestion
public class FruitTransaction {
```

Reply:
refactored
but Transaction is synonym to Negotiation in vocabulary